content/docs/2_cookbook/9_setup/0_kirby-meets-purgecss/cookbook-recipe.txt

```diff
@@ -0,0 +1,151 @@
+Title: Kirby meets PurgeCSS
+
+----
+
+Published: 2021-08-09
+
+----
+
+Text:
+
+This is a walkthrough on how to set up and use PurgeCSS with a Command-line interface (CLI) or with a Graphical user interface (GUI). (link: https://purgecss.com/ text: PurgeCSS) is a tool to remove unused CSS and it can be part of your development workflow.
+
+<info>
+When you are building a website, you might decide to use a CSS framework like TailwindCSS, Bootstrap, Bulma, MaterializeCSS, Foundation, etc... But you will only use a small set of the framework, and a lot of unused CSS styles will be included.
+</info>
+
+> This is where PurgeCSS comes into play. PurgeCSS analyzes your content and your CSS files. Then it matches the selectors used in your files with the ones in your content files. It removes unused selectors from your CSS, resulting in smaller CSS files.
```

Comment:
Is there a reason why this is formatted as quote? If it was a quote, I think it should give the source. But for readability's sake, I'd convert this into to a standard paragraph nonetheless.

Reply:
It´s a copy/quote from https://purgecss.com/, but you´re right. With an additional line to mention the author, it looks odd.